src/test/java/in/balendin/schoolmanagment/testcases/features/AddOrganizationTestCase.java

```diff
@@ -0,0 +1,57 @@
+package in.balendin.schoolmanagment.testcases.features;
+import in.balendin.schoolmanagment.steps.AddOrganizationSteps;
+import in.balendin.schoolmanagment.steps.LoginSteps;
+import in.balendin.schoolmanagment.testcases.basetest.BaseTestCase;
+import net.serenitybdd.junit.runners.SerenityRunner;
+import net.thucydides.core.annotations.*;
+import org.junit.Before;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.MethodSorters;
+
+@Narrative(text = {"As a super admin,", "I want to be able to Create Organization"})
+@RunWith(SerenityRunner.class)
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+@WithTagValuesOf({"addOrganization", "regression"})
+
+public class AddOrganizationTestCase extends BaseTestCase {
+
+    @Steps
+    LoginSteps loginSteps;
+    @Steps
+    AddOrganizationSteps organizationSteps;
+
+
+    @Before
+    public void setUp() {
+        loginSteps.navigateToApplication();
+        loginSteps.loginAsSuperAdmin();
+    }
+
+    @Test()
+    @Title("verify super admin can create Organization Successfully")
+    public void T1_addOrganization() {
```

Comment:
@rajaYendluri please remove the prefix name for every method name

Reply:
completed